torch_geometric/explain/algorithm/subgraphX_explainer.py

```diff
@@ -0,0 +1,646 @@
+import copy
+import logging
+import math
+from collections import Counter
+from functools import partial
+from os.path import isfile, join
+from typing import Callable, Dict, List, Optional, Union
+
+import networkx as nx
```

Comment:
What is the specific need for networkx? If it is only for subgraph and number of nodes (my current understanding, correct if wrong), then I would suggest using Pytorch/torch_geometric features instead.